### DISCUSSION TITLE
Please help me to integrate Billing system

### DISCUSSION THREAD

**Discussion by secureweb16** [Kyon147/laravel-shopify#208] [General]
Hii @Kyon147 , 

Could you please let me know how to integrate the billing system into Shopify public app or do you have any dummy setup in Laravel if yes then please share,
Currently don't have any idea how to handle recurring monthly payments..

**Comment by secureweb16** [comment]
Also please let me know if the package itself handle recurring payment, If not then please let me know how to recurring charge handle any helper function available.

**Reply by rvibit** [reply]
Did you check this documentation? https://github.com/Kyon147/laravel-shopify/wiki/Creating-a-Billable-App
